tools/Replay.hpp

```diff
@@ -216,32 +267,104 @@ struct Stats {
         dumpMap(_recvd);
     }
 
-    void dumpStressToCSV(size_t runMs)
+
+    //loading phase will record the set up times and loaded phase records the document loading times
+    void perfSetupLoadEditPhases(std::string phase)
```

Comment:
So - I guess, it's perhaps a better self documenting API to have an enumeration of phase type; there are some nice macros Ash has that allow you to get a nice string name from each Enum value - we can then use as the string for that; and then it's more clear what the phases are and that we move through them; checkout: common/Log.hpp and the STATE_ENUM macro =)

Reply:
I've tried to implement this and it does work but i'm unsure if this is the correct way to implement the Enum value?

Reply:
Looks lovely =)